.travis.yml

```diff
@@ -16,7 +16,7 @@ before_script:
         if [ "${TRAVIS_EVENT_TYPE}" = "cron" ]; then
             PYTEST_ADDOPTS="-m 1"
```

Comment:
`-m normal and functional and functional_long_running` is better than `-m 1`?

Reply:
Probably yes, because it's obvious what that means.

Reply:
ok I will change for this.